elasticsearch/client/utils.py

```diff
@@ -26,6 +26,10 @@ def _escape(value):
     elif isinstance(value, bool):
         value = str(value).lower()
 
+    # decode bytestrings
+    elif isinstance(value, bytes):
+        value = value.decode()
```

Comment:
if it's `bytes` then we don't want to `decode` it, just pass it through to avoid any problems with en/decoding and the extra work.